server/scheduler/GeneticAlgorithm.py

```diff
@@ -5,7 +5,7 @@
 import random as rnd
 import datetime  # for time comparison
 import time
-from .sql import getGATAs
+from .sql import getGATAs, getCourses, getLabs
```

Comment:
Importing new classes to pull getCourses and getLabs into GA.py file to pull data from database by calling these functions